backend/tests/test_performance_scalability.py

```
@@ -0,0 +1,266 @@
+- Test 5.1: Broadcast completes within latency target (<50ms for 100 connections)
+- Test 5.2: Broadcast handles large message payloads (>1MB)
+- Test 5.3: Memory usage remains stable during broadcasts
+- Test 5.4: Broadcast throughput scales with connection count (linear scaling)
+"""
+
+import asyncio
+import json
+import time
+import psutil
+import pytest
+from unittest.mock import AsyncMock, MagicMock, patch
```

Ruff (0.12.2)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)